cdm/core/src/test/java/ucar/nc2/internal/ncml/TestNcmlModifyAtts.java

```diff
@@ -258,4 +268,70 @@ public void testReadData2() throws IOException {
     Assert2.assertNearlyEquals(dataI.getDoubleNext(), 4.0);
     Assert2.assertNearlyEquals(dataI.getDoubleNext(), 2.0);
   }
+
+  @Test
```

Comment:
Would you mind refactoring these to have a helper method that takes the filename and does all the verification to avoid duplication? :) Otherwise looks great

Reply:
Yeah, I just refactored and pushed it again. Thanks for the input! I didn't catch that they ended up being the same. Let me know what you think :)